interfaces/extension/ExtensionItem.d.ts

```diff
@@ -1,4 +1,4 @@
-import { Feature } from "./Feature";
+import { ExtensionFeature } from "./Feature";
```

Comment:
The code failed to transpile because "Feature" was missing. Let me know if this was intentional so I can revert the diff